### DISCUSSION TITLE
Transcribing multilingual court hearings (Canada)

### DISCUSSION THREAD

**Discussion by simon-lawyer** [openai/whisper#655] [Show and tell]
Hi there,

I just used Whisper to transcribe 100+ Supreme Court of Canada hearings. Take a look: obiter.ai/scc/.

Really impressive stuff. Thank you!

**Comment by expdis** [comment]
Hi Simon,

Can you please share how you link whisper, pyannote and SpeechBrain to transcribe?  Thanks!

**Comment by RYucel** [comment]
I just have similar project and very hard to get clear speech because of noise, cracks, mic hittings, inaudible speaks by judges...I am really struggling. How can you manage it? Good project by the way

**Reply by mliradelc** [reply]
Maybe you can try NoiseTorch https://alternativeto.net/software/noisetorch/about/ or [Noise Suppression for voice](https://github.com/werman/noise-suppression-for-voice)

**Reply by rahulbansal16** [reply]
I want to build a real-time transcription for my product. My approach is to use VAD to detect the silences and then send those chunks to Whisper for transcription. But I've seen that Whisper has a higher word error rate compared to Assembly AI, and my worry is if I make it real-time, the word error rate is going to increase. Any suggestions you have to get very good accuracy for particular domains?
